packages/manager/src/components/FileUploader/FileUploader.styles.ts

```diff
@@ -0,0 +1,88 @@
+import Button from 'src/components/Button';
+import { styled } from '@mui/material/styles';
+import { isPropValid } from 'src/utilities/isPropValid';
+import Typography from 'src/components/core/Typography';
+import { makeStyles } from 'tss-react/mui';
+import { Theme } from '@mui/material/styles';
+
+export const StyledDropZoneDiv = styled('div')(({ theme }) => ({
```

Comment:
```suggestion
export const StyledDropZoneDiv = styled('div', { label: 'StyledDropZoneDiv' })(({ theme }) => ({
```

Let's please add labels for all styled components. Makes it easier to debug in the browser!